app/create_file.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
+from sys import argv
+from os import mkdir, chdir
+
+filename = ""
+directories = []
+
+if "-f" in argv:
+    index = 0
+
+    for _ in range(len(argv) - 1):
```

Comment:
Changing the global directory with chdir without guaranteeing restoration at the end is dangerous.
Fix: Use try-finally to ensure the current directory is restored. This avoids side effects in the execution environment, especially in scripts that depend on the current directory.